--- ../atl.ml	2015-05-12 19:56:33.650443534 +0800
+++ atlweb.ml	2015-05-12 19:56:32.606454349 +0800
@@ -17,5 +17,12 @@
 end
-
-let bool2str b = if b then "Y" else "n"
-let printf= Printf.printf
+;;
+let print_string s = ()
+let print_endline s = ()
+let print_newline () = ()
+(*let caml_ml_output_char c = ()*)
+let print_char c = ();;
+
+let bool2str b = if b then "Y" else "n";;
+let output_buffer = Buffer.create 1000;;
+let printf fmt = Printf.bprintf output_buffer fmt;;
 
@@ -184,8 +191,4 @@
 
-let phi =
-	if Array.length Sys.argv > 1
-	then Formula.of_string Sys.argv.(1)
-	else Formula.of_string "({1}p&{1}~p)";;
-	(*else UNTIL (ATOM 'p', (AND (ATOM 'p', NOT (ATOM 'p')))) *)
-let use_weak = false;;
+let phi = Formula.of_string (Js.to_string (Dom_html.window##location##search))
+let use_weak = true;;
 
@@ -697,2 +700,2 @@
 
-printf "Finished Processing %s\n" (Formula.to_string phi);
+Dom_html.window##alert (Js.string (  result ));;
